main.py

```diff
@@ -7,68 +7,64 @@
 from loguru import logger
 
 load_dotenv()
-ATTEMPTS_TO_POST_GAZETTES = 3
 
 
 def tweet(message: str):
     auth = tweepy.OAuthHandler(os.getenv("CONSUMER_KEY"), os.getenv("CONSUMER_SECRET"))
     auth.set_access_token(os.getenv("ACCESS_TOKEN"), os.getenv("ACCESS_TOKEN_SECRET"))
 
     api = tweepy.API(auth)
-    api.update_status(message)
-    logger.info("The gazette was posted on twitter!")
+    try:
+        api.update_status(message)
+        logger.info("The gazette was posted on twitter!")
+    except tweepy.TweepError as e:
+        logger.exception(e)
 
 
-def refresh_maria_quiteria_api_token():
+def create_maria_quiteria_api_token():
     headers = {
         "content_Type": "application/json"
     }
     url = f"{os.getenv('MARIA_QUITERIA_API_HOST')}/token/"
     data = {
-        "username":f"{os.getenv('USERNAME_CREDENCIALS')}",
-        "password": f"{os.getenv('PASSWORD_CREDENCIALS')}"
+        "username":f"{os.getenv('MARIA_QUITERIA_USERNAME_CREDENCIALS')}",
+        "password": f"{os.getenv('MARIA_QUITERIA_PASSWORD_CREDENCIALS')}"
     }
 
-    refresh_token_response = requests.post(url, headers=headers, data=data)
-    if refresh_token_response.status_code == 200:
-        logger.info('New token created')
-        new_token = f"Bearer {refresh_token_response.json()['access']}"    
+    token_response = requests.post(url, headers=headers, data=data)
+    logger.info('Getting token from Maria Quitéria')
 
-    return new_token
+    if token_response.status_code == 200:
+        new_token = f"Bearer {token_response.json()['access']}"
```

Comment:
Pode retornar direto aqui :)
```suggestion
        return f"Bearer {token_response.json()['access']}"
```